### PR TITLE
(reopen) change highlighting of static to match block, if

### PR DESCRIPTION
**Pull request by morganholly**
see #86

### CODE REVIEW

File: syntaxes/nim.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
     },
     {
       "comment": "Keywords that affect program control flow or scope.",
-      "match": "\\b(block|break|case|continue|do|elif|else|end|except|finally|for|if|raise|return|try|when|while|yield)\\b",
+      "match": "\\b(block|break|case|continue|do|elif|else|end|except|finally|for|if|raise|return|try|when|while|yield|(static(?= *:))|of)\\b",
```
Comment (saem) [inline syntaxes/nim.json:151]: Shouldn't only the static block one count here? Wouldn't that also match it in types for procedure parameters?

Comment (morganholly) [reply syntaxes/nim.json:151]: ah i thought i took care of that but i'll look more into the places where static is used and make sure it doesn't match those as well

Comment (morganholly) [reply syntaxes/nim.json:151]: ok so it matches a colon after the static. a type like `static string` would have a colon before not after

**Comment by saem** [comment]
@morganholly looks like you've had quite the adventure. The revert via a revert commit isn't what I meant but a history rewrite to drop the commit.

Looks like a all the commits aren't actually material as some of them are likely in main. This article will walk you through interactive rebase and how to remove unnecessary commits: https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d

Then if there is an issue with conflicts that's ok, I'll guide you through that with some instructions.

**Comment by morganholly** [comment]
for a change this small, maybe it would be better to make a new branch from the repo as it is currently bringing in the changes in a new commit

**Comment by morganholly** [comment]
ok i created #91

**Pull request closed without merging**